network/model.go

```diff
@@ -1,6 +1,6 @@
-package server
+package network
 
-type CreatePlayerData struct {
+type CreatePlayerModel struct {
```

Comment:
🔮 📖 Would `CreatePlayerRequest` be a more appropriate name?

Reply:
👍